src/routes/MainRouter.tsx

```diff
@@ -16,7 +16,7 @@ export const MainRouter: React.FC = () => {
     <BrowserRouter basename={process.env.PUBLIC_URL}>
       <Routes>
         {/* TODO: based on user profile, redirect to patient, dashboard or whatever */}
-        <Route index element={<Navigate to="patients" replace />} />
+        <Route index element={<Navigate to="/dashboard" replace />} />
```

Comment:
Restore default navigation to /patients.
Add a link to dashboard in AppHeader menu